### DISCUSSION TITLE
有兼容xp系统的版本吗？

### DISCUSSION THREAD

**Discussion by bige47** [Blinue/Magpie#452] [Q&A]
我是很希望有个xp系统能运行的版本，因为许多老游戏都只能在xp系统上运行，画质实在太差了

**Comment by Blinue** [comment]
不能用于 xp/win7，Magpie 是面向 Win10/11 设计的。另外 Win10/11 的兼容性很好，居然有不能运行的老游戏吗？

**Reply by hooke007** [reply]
主要是上古的、冷门的，还有些是因为crack补丁没有支持win10的。
要说热门的话，红警2就是，打了社区开发的补丁（我之前在某个issue提到的[cnc-ddraw](https://github.com/FunkyFr3sh/cnc-ddraw)）才能接近完美跑起来。
我遇到常见的是需要(win)SxS.dll的galgame，不过直接copy到目录就能跑起来了。

**Comment by BLACKCHARGE** [comment]
实际上只要按照微软的标准构建没有多少游戏是只能在XP上运行不能在win10上运行的，至少我没咋听说过，就算不能直接运行网上找找也基本上有能运行的解决方案（打补丁，运行库，兼容性模式运行），出问题的主要是各类游戏的汉化补丁以及破解补丁上面，这种私人开发的东西很多因为构建的不标准会出现运行不了的情况，后者可以正版解决，前者一个可能解决方案是用虚拟机全屏，然后直接放大虚拟机窗口。

**Reply by BLACKCHARGE** [reply]
magpie使用的directx11图形接口，win7可能还有戏，xp不支持dx11，应该是没有使用的希望的

**Comment by BangrongMo** [comment]
可以试试虚拟机里面运行，然后放大虚拟机
